src/main/proto/uri.proto

```diff
@@ -51,7 +125,7 @@ message UriRequest {
 
 // Return value for the lookupUriRequest
 message UriResponse {
-  repeated string uris = 1;  // List of qualified (absolute) uris
+  repeated UUri uris = 1;  // List of qualified (absolute) uris
```

Comment:
better naming please! I am so confused as to what these are used for

Reply:
To this note, we added UuriRequest and UUriResponse but never used anything but the URI so I'll remove them and update UDiscovery.